internal/gateway/gateway_test.go

```diff
@@ -29,75 +29,17 @@ import (
 	"testing"
 	"time"
 
+	"github.com/ForgeRock/iot-edge/v7/internal/mocks"
+
```

Comment:
Is this gap intentional?

Reply:
No, it's just a result of running two different import sorting types. I'll fix it up.